static/js/common.js

```diff
@@ -1,3 +1,8 @@
+// This reloads the module in development rather than refreshing the page
+if (module.hot) {
```

Comment:
Since I expect we will be adding something like this to more than one file, I just stubbed the relevant module in zjunit.